src/WinUI/ArcGIS.WinUI.Viewer/Samples/Data/ManageFeatures/ManageFeatures.xaml

```diff
@@ -0,0 +1,28 @@
+<UserControl 
+    x:Class="ArcGIS.WinUI.Samples.ManageFeatures.ManageFeatures"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:esri="using:Esri.ArcGISRuntime"
+    xmlns:esriUI="using:Esri.ArcGISRuntime.UI.Controls">
+    <Grid>
+        <esriUI:MapView x:Name="MyMapView" />
+        <Border Style="{StaticResource BorderStyle}">
```

Comment:
Totally your choice - here and in WPF, you could try shrinking the border width a bit.